spinoso-string/src/lib.rs

```diff
@@ -1544,7 +1544,7 @@ impl String {
         if self.is_empty() {
             return false;
         }
-        let bytes_to_remove = if self.inner.as_slice().ends_with(b"\r\n") {
+        let bytes_to_remove = if self.inner.ends_with(b"\r\n") {
```

Comment:
I did not add additional tests because existing doc tests already cover this 'if' branch. If I change the following the doc test for `String#chop` would fail.
```diff
-        let bytes_to_remove = if self.inner.ends_with(b"\r\n") {
+        let bytes_to_remove = if self.inner.ends_with(b"abc\r\n") {
```

Reply:
awesome thanks for verifying.